src/platform/user/authentication/components/DowntimeBanner.js

```diff
@@ -38,4 +40,22 @@ const downtimeBanners = [
   },
 ];
 
-export default downtimeBanners;
+export const DowntimeBanner = ({ dependencies, headline, status, message }) => (
+  <ExternalServicesError dependencies={dependencies}>
+    <div className="downtime-notification row">
+      <div className="columns small-12">
+        <div className="form-warning-banner">
+          <va-alert headline={headline} visible status={status}>
+            {message}
+          </va-alert>
```

Comment:
Implement the same slot paradigm for consistency sake.

Reply:
💯  good catch